tensorflow/python/ops/rnn_cell.py

```diff
@@ -85,9 +85,11 @@ def _state_size_with_prefix(state_size, prefix=None):
 
 
 class RNNCell(object):
-  """Abstract object representing an RNN cell.
+  """Array of RNN cells.
```

Comment:
I was only trying to make the usage match the definition used by the rest of the community, including papers linked to in this file.  I'll change it as per your request, though.